style(*): grammar, spelling, punctuation (#539)
discordjs/guide: guide/command-handling/adding-features.md

+       }
+
+       return message.channel.send(reply);
	}
```

> Use an `if` statement to check if the `usage` property exists (and is truthy) first. For example, if you forget to provide the property, it would display `undefined` in the reply string. A simple precaution such as this can significantly improve the user experience.

Reviewer: ```suggestion
Use an `if` statement to check if the `usage` property exists and append to the reply string if it does.
```

Now when you try to use the kick command inside a DM, you'll get the appropriate response, preventing your bot from throwing an error.

<div is="discord-messages">
	<discord-message author="User" avatar="djs">
		!kick
	</discord-message>
	<discord-message author="Tutorial Bot" avatar="blue" :bot="true">
		I can't execute that command inside DMs!
	</discord-message>
</div>

## Cooldowns

Spam is something you generally want to avoid–especially if one of your commands requires calls to other APIs or takes a bit of time to build/send. Cooldowns are also a very common feature bot developers want to integrate into their projects, so let's get started on that!

You'll be using the ping command to test this on. Add in the following bit of code:

```diff
module.exports = {
	name: 'ping',
+   cooldown: 5,
	execute(message) {
		message.channel.send('Pong.');
	},
};
```

> The `cooldown` property is the amount (in seconds) that the user will have to wait for before using that command again. You'll be using Collections again to store what you need.

Reviewer: ```suggestion
The `cooldown` property is the number of seconds the user must wait before being able to execute that command again. You'll be using Collections again to store what you need.
```
